StackExchange.Redis/StackExchange/Redis/GeoUnit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// Units associated with Geo Commands
+    /// </summary>
+    public enum GeoUnit : short
+    {
+        /// <summary>
+        /// Meters
+        /// </summary>
+        m,
```

Comment:
The enum names of been updated, and a ```private string[]``` member has been implemented to translate to the redis accepted value.